templates/notes_app_shelf/TEMPLATE.md

```diff
@@ -0,0 +1,126 @@
+---
+name: Flutter + Firebase Auth + Shelf Backend
+description: Build a Notes App using Flutter Web, Firebase Auth and a custom backend using Shelf.
+tags: ["firebase", "shelf", "flutter", "firestore"]
+username: Invertase
+---
+
+# Flutter + Firebase Auth + Shelf Backend
+
+## Overview
+
+Build a Simple Notes App using Flutter Web, Firebase Auth and Dart for our backend.
+
+### Features
+
+- [Shelf](https://pub.dev/packages/shelf)
+- [Flutter](https://flutter.dev)
+- [Firebase](https://firebase.google.com/)
+
+### Getting Started
+
+#### Bootstrap
+
+Initialize your project using the `notes_app_shelf`
+
+```shell
+$ globe create -t notes_app_shelf
+```
+
+#### Setup Firebase Project
+
+You must create a Firebase account, generate Firebase Admin Service Account and download.
+
+To generate a private key for your service account:
+
+- In the Firebase console, open **Settings** > **[Service Accounts](https://console.firebase.google.com/project/_/settings/serviceaccounts/adminsdk)**
+- Click **Generate New Private Key**, then confirm by clicking **Generate Key**
+- Securely store the JSON file containing the key.
+
+#### Setup Local Variables
+
+- Run this command with the path to the service account file.

```

Comment:
this command will be needed on every terminal session. Better to have it in `.bashrc` or `.zshrc`

Reply:
Resolved.